single_stage_detector/ssd/README_Inference.md

```diff
@@ -0,0 +1,46 @@
+# Inference
+
+## Training
+
+The model was trained on coco2017 dataset using 8xV100 GPUs and FP32 with the following command:
+
+```bash
+python -m torch.distributed.launch --nproc_per_node=8 --use_env train.py \
```

Comment:
The training WG will decide the hparam rules sometime before the 2.0 freeze. Having this discussion now is pointless since we might switch to OpenImages.